wallet/lib/pages/detailed_asset_view/widgets/tab_fields.dart

```diff
@@ -204,16 +216,8 @@ class _TabFieldState extends State<TabField> {
       child: Row(
         children: [
           Text(
-            value.substring(0, 6),
-            style: TextStyle(color: AppColors.kWhite),
-          ),
-          Text(
-            "...",
-            style: TextStyle(color: AppColors.kWhite),
-          ),
-          Text(
-            value.substring(value.length - 5, value.length),
-            style: TextStyle(color: AppColors.kWhite),
+            value.substring(value.length - 10, value.length),
```

Comment:
Whats the purpose for -10?

Reply:
So it was showing 5 character before, 5 characters seems very least.

<img width="278" alt="image" src="https://user-images.githubusercontent.com/115459890/201718035-da8034dd-8535-4c49-bbc8-41d5e3e4ddfe.png">


So i added 10 characters here.. 


<img width="278" alt="image" src="https://user-images.githubusercontent.com/115459890/201721290-1657d428-9725-4754-a6be-d3670f21d3cc.png">

Reply:
What if the string is "Ja" what will happen?

Reply:
an exception will come.. but these are the addresses and they are long.. if you want me to remove this i will remove it.